doc/4.data.structures/01.scalars.pd

```diff
@@ -53,7 +53,7 @@
 #X text 544 10 IMPORTANT NOTE:, f 63;
 #X obj 162 286 filledpolygon 250 q 5 0 0 20 z 40 0;
 #X text 544 350 The reason you probably want to do this is because it is more intuitive to have an increase in "Y" values reflect on an increase in pixels moving upwards. This is specially important for arrays \, but it also affects the scalars in this example. For the triangular shapes \, the apex value depends on a 'z' field which is a vertical coordinate. Positive values of 'z' move the apex upwards \, while negative values move it downwards., f 63;
-#X text 22 145 When you right click on a scalar for properties \, you can see the template namem the specified field names and then the scalar's data \, which is the template name followed by the field values., f 70;
+#X text 22 145 When you right click on a scalar for properties \, you can see the template name\, the specified field names and then the scalar's data \, which is the template name followed by the field values., f 70;
```

Comment:
quoting from the PR description:
>  (as a general rule-of-thumb we shouldn't include changes to Pd-files in this PR, as it is just too easy to end up with unresolvable file conflicts)

please direct documentation updates to #2042 (or whatever the current "documentation updates" branch by @porres is

Reply:
thats the one

Reply:
Ah, sorry, forgot about that. Just removed the commit and asked @caitp to make a PR to https://github.com/porres/pure-data/tree/Documentation.

Reply:
I have to say that the current situation is a bit confusing. Can't we have a documentation branch in the main repo (and give @porres write access)? If someone makes a PR, I can just change the base branch, instead of asking them to make a new PR to some fork...